submissions/sprint_a/helpers.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+import random
+import csv
+
+
+class FileManager:
+    """Handle local file system IO."""
+
+    @staticmethod
+    def get_extension(path):
+        """Get file extension from file path."""
+        return os.path.splitext(path)[-1][1:]
+
+    @staticmethod
+    def read_json(path, mode='r', *args, **kwargs):
+        """Reads the file and returns its contents."""
+        with open(path, mode=mode, *args, **kwargs) as handle:
+            return json.load(handle)
+
+
+class Vocabulary:
+    """Standardize vocabulary representation from multiple sources."""
+
+    files = FileManager()
+
+    @classmethod
+    def from_file(cls, path, *args, **kwargs):
+        """Takes a path determines its extension and returns the files data"""
+        extension = cls.files.get_extension(path)
+        if extension == 'csv':
+            representation = cls.csv_reader(path)
+        else:
+            representation = cls.strategies(extension)(path, *args, **kwargs)
+        return representation
+
+    @classmethod
+    def from_json(cls, path, fields=True, *args, **kwargs):
+        """Takes a path and returns the files data"""
+        data = cls.files.read_json(path, *args, **kwargs)
+        return data
+
+    @classmethod
+    def strategies(cls, file_extension, intent='read'):
+        """Takes a json file and returns an object"""
+        input_strategies = {'json': cls.from_json}
+        if intent is 'read':
+            return input_strategies[file_extension]
+
+    @classmethod
+    def csv_reader(cls, path):
+        """Takes the given path and returns the CSV data"""
+        csvfile = open(path, 'r')
+        words = csv.reader(csvfile, delimiter=',')
+        vocabulary = {'vocabulary': []}
+        line = 0
+        for row in words:
+            if line == 0:
+                line += 1
+                continue
+            else:
+                vocabulary['vocabulary'].extend(row)
+
+        vocabulary['vocabulary'] = [v for v in vocabulary['vocabulary'] if v != '']
+        return vocabulary
+
+
+class EpithetGenerator:
+    path = """
+       /Users/aj/Documents/Back-End/backend-epithet-generator/resources/data.json
```

Comment:
Use relative paths or absolute paths built by joining PROJECT_ROOT defined in init file.